### ISSUE TITLE
multiple links in single property

### ISSUE THREAD

**Issue by chonton** [fabric8io/docker-maven-plugin#558]
Our corporate parent pom configures docker plugin and sub-projects use the docker packaging.  We'd like to have each sub-project have different <links>.  Unfortunately, it's rather difficult to provide just the <links> configuration in the sub-project.  I cannot figure any combination of combine.children and combine.self that doesn't require re-configuring large sections of the image/run element.  What I have done currently is set a single link to a property which is defined in the sub-project.

I now need multiple links.  I'd like to either add a new image/run element called linksList the takes a comma/space separated list of link declarations OR modify <link> to support multiple declarations.  Or, do you have other ideas how to add multiple links in a template fashion? 

I'm will work on pull request based upon your suggested direction.

**Comment by rhuss** [comment]
You mean to add `<link>` elements to the list of `<links>` ? Do you have a concrete example for me ? (and sorry for the late answer)

**Comment by chonton** [comment]
Currently, I need to write:

```xml
<links>
  <link>mongo:db</link>
  <link>identity</link>
</links>
```

I would like to be able to write in parent pom:
```xml
<linksList>${service.dependencies}</linksList>
```
And in project module, service.dependencies will be set:
```xml
<properties>
  <service.dependencies>mongo:db,identity</service.dependencies>
</properties>
```

**Comment by rhuss** [comment]
Thanks for the PR !

I'm just thinking whether it wouldn't be better to reuse `<links>` for this feature in order to keep the configuration focussed. The idea is to allow

```xml
<links>
   <link>mongo:db,identity<link>
</links>
```

which would work nice if `,` is not allowed as part of a network name. That way we can also group nicely links, since we can provide multiple `<link>` entries, each comma separated.

wdyt ?

**Comment by chonton** [comment]
Cool, I'll modify PR.

**Comment by rhuss** [comment]
Thanks ! Its now included in the latest 0.16.8 release.
